ESCPOS_NET/ESCPOS_NET.csproj

```diff
@@ -24,7 +24,8 @@ Allows creating receipts with all common functionality supported.</Description>
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="5.0.0" />
-    <PackageReference Include="SixLabors.ImageSharp" Version="1.0.0" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="1.0.4" />
+    <PackageReference Include="SixLabors.ImageSharp.Drawing" Version="1.0.0-beta13" />
```

Comment:
Just adding `SixLabors.ImageSharp` won't get us `SixLabors.ImageSharp.Drawing`? Without C# knowledge, I am asking this.

Reply:
[`Drawing` is a separate package](https://www.nuget.org/packages/SixLabors.ImageSharp.Drawing/1.0.0-beta14).